src/usr/local/www/head.inc

```diff
@@ -461,8 +461,32 @@ if (($pagename === "index.php") && ($numColumns > 2)) {
 	}
 ?>
 		<ul class="context-links">
-<?php
 
+	<?php if ($dashboard_available_widgets_hidden): ?>
+		<li>
+			<a onclick="$('#widget-available').toggle(360);" title="<?=gettext("Show/Hide available widgets panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Widgets</i>
+			</a>
```

Comment:
Perhaps this doesn't need to be a button, but just (bold) text. That would eliminate the special styling because it would inherit the title bar styling I think ??